docs: PEM-10477: Known Issue Version Split (4.6) (#10126)
spectrocloud/librarium: docs/docs-content/troubleshooting/vmo-issues.md

- Beginning with Palette version 4.5.20, self-hosted environments installed with the
  [Palette CLI](../automation/palette-cli/palette-cli.md).

- Beginning with Palette version 4.6.23, self-hosted environments installed with the
  [Palette Helm chart](../enterprise-version/install-palette/install-on-kubernetes/install-on-kubernetes.md).

:::

1. Log in to your
   [self-hosted Palette](../enterprise-version/system-management/system-management.md#access-the-system-console) or
   [Palette VerteX](../vertex/system-management/system-management.md#access-the-system-console) system console.

2. From the left main menu, select **Enterprise Cluster**.

3. On the **Overview** tab, download the **Kubernetes Config File**.

4. From your terminal, set the `KUBECONFIG` variable to the file path of the kubeconfig.

   ```shell
   export KUBECONFIG=<path-to-kubeconfig>
   ```

5. Check the current `limit-rps` annotation on the `hubble-foreq-ingress-resource` Ingress resource.

   ```shell
   kubectl get ingress hubble-foreq-ingress-resource --namespace hubble-system \
   --output jsonpath='{.metadata.annotations.nginx\.ingress\.kubernetes\.io/limit-rps}'
   ```

>    If the value is `10` or another low number, update it to `10000`.

Reviewer: This is a bit vague, can we give them the exact values?

>    If the value is `10` or another low number, update it to `10000`.

Author: ```suggestion
   If the value is lower than `10000`, update it.
```
